src/main/java/seedu/task/model/task/Effort.java

```diff
@@ -0,0 +1,52 @@
+package seedu.task.model.task;
+
+import java.util.Objects;
+
+import seedu.task.model.task.exceptions.InvalidEffortException;
+
+/**
+ * Represent a Task's required effort in the task book.
+ */
+public class Effort {
+    public static final String MESSAGE_CONSTRAINTS = "Effort must be valid";
+    private long effort;
+
+    /**
+     * Creates an effort instance which is an abstraction on the estimated effort required to complete a task.
+     * @param effort
+     */
+    public Effort(long effort) {
+        if (effort < 0) {
+            throw new InvalidEffortException();
+        }
+        this.effort = effort;
+    }
+
+    public Effort() {
+        this.effort = 2;
+    }
```

Comment:
How about a value that users can identify better with? I think from a user standpoint they might be curious why 2 was chosen.